pages/types-torture-different-residence.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1>
+          Types of Torture at Different Residence Locations
+          <div
+            class="inline-flex items-center justify-center align-middle w-6 h-6 hover:w-7 hover:h-7 mb-1.5 ml-1 cursor-pointer"
+          >
+            <img
+              src="/images/infoIcon.svg"
+              class="w-full h-full pt-0.5"
+              aria-label="Page Information Popup"
```

Comment:
I am getting a strange bit of floating text that says page information. Suspect it is because of changes we've made in earlier branches but flagging to you to take a look. Thanks Ruby

Reply:
yeah this is because visibility hidden is in the layouts css which hasn't been merged in yet, should i add it in to this branch or wait till merge?

Reply:
ahh makes sense. I will merge in nuxt3-dev now as I'm now up to date. Will see how it looks after this. Thank you

Reply:
Looks good now so merging in. Thanks Ruby